backend/app/agents/devrel/github/services/github_mcp_server.py

```
@@ -0,0 +1,96 @@
+            raise HTTPException(status_code=500, detail="GitHub service not available")
+        
+        result = github_service.repo_query(request.owner, request.repo)
+        
+        if "error" in result:
+            return RepoInfoResponse(status="error", data={}, error=result["error"])
+        
+        return RepoInfoResponse(status="success", data=result)
+        
+    except Exception as e:
+        logger.error(f"Error getting repo info: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)